app/controllers/posts_controller.rb

```diff
@@ -24,6 +30,11 @@ def create_comment
     end
   end
 
+  def destroy_comment
+    Comment.find(params[:comment]).destroy
+    redirect_to post_path
+  end
+
```

Comment:
- [ ] Please create a Comment Controller file `comments_controller.rb` to manage all your comment actions ( `create_comment`, `destroy_comment`).

- [ ] Please create a Like Controller file `like_controller.rb` to manage all like actions (`create_like`)